clients/client/src/test/java/org/projectnessie/client/auth/TestBasicAuthProvider.java

```diff
@@ -126,7 +127,7 @@ void checkAuth(NessieAuthentication authentication) {
     assertThat(authFilter[0]).isInstanceOf(RequestFilter.class);
 
     HttpHeaders headers = new HttpHeaders();
-    RequestContext context = new RequestContext(headers, null, null, null);
+    RequestContext context = new RequestContextImpl(headers, null, null, null);
```

Comment:
nit: looks odd to call a constructor with so many nulls (when usually method etc are not suppsoed to be null)...
maybe this and httpheaders above should be in some test utils as `newDummyRequestContext()` or the like (used in multiple places)